test/wallet-rpc-test.js

```diff
@@ -318,6 +318,44 @@ describe('Wallet RPC Methods', function() {
     });
   });
 
+  it('should rpc abortRescan', async () => {
+    assert.strictEqual(wdb.height, 104);
+    assert.strictEqual(wdb.height, node.chain.height);
+
+    // When rescanning is not triggered
+    const response = await wclient.execute('abortrescan');
+    assert.strictEqual('rescanning is not running; abort does not make sense', response);
```

Comment:
First, the error case can be own its own test. ie. `it('should not abort rescan if not rescanning...`

and also:

```js
await assert.rejects(
  wclient.execute('abortrescan'),
  {message: 'WalletDB is not rescanning.'}
);
```